arxivdigest/frontend/templates/profile.html

```diff
@@ -69,38 +69,18 @@ <h3>{{ user.firstname }}&nbsp{{ user.lastname }}'s profile information:</h3>
                     </tr>
                     <tr>
                         <td>Notification Interval:</td>
-                        <td>{{ user.notification_interval }}</td>
-                    </tr>
-                    <tr>
-                        <td>
-                            Personal data:
-                        </td>
-                        <td>
-                            <a href="{{ url_for('general.download_personal_data') }}">
-
-                                Download my personal data
-                            </a>
-                        </td>
-                    </tr>
-                    <tr>
-                        <td>
-                            <a href="{{ url_for('general.passwordChange') }}">
-                                <div class="btn btn-primary">
-                                    Change password
-                                </div>
-                            </a>
-                        </td>
-                        <td>
-                            <a href="{{ url_for('general.modify') }}">
-                                <div class="btn btn-primary">
-                                    Modify profile
-                                </div>
-                            </a>
-                        </td>
+                        <td>{% if user.notification_interval == 1 %}daily{% else %}weekly{% endif %}</td>
                     </tr>
                     </tbody>
                 </table>
             </div>
+            <div class="col-md-4">
+                <ul class="list-group">
+                    <li class="list-group-item"><a href="{{ url_for('general.passwordChange') }}"><span class="glyphicon glyphicon-user" aria-hidden="true"></span> Modify profile</a></li>
```

Comment:
Modify profile should lead to modify profile endpoint

Reply:
Well spotted. Fixed.